src/components/LabelMapper/ArtifactQualityLabelMapper.tsx

```diff
@@ -50,8 +41,6 @@ interface IArtifactQualityLabelProps {
   quality: Artifact['artifactQuality'];
 }
 
-export const ArtifactQualityLabel = ({ quality }: IArtifactQualityLabelProps) => {
-  const artifactQuality = ARTIFACT_QUALITIES[quality];
-
-  return artifactQuality ? <Label color={artifactQuality.color}>{artifactQuality.text}</Label> : <EmptyStateSymbol />;
-};
+export const ArtifactQualityLabel = ({ quality }: IArtifactQualityLabelProps) => (
```

Comment:
`ArtifactQualityLabelMapper` and `IArtifactQualityLabelMapperProps`,, apply the same everywhere

Reply:
Please I would like to ask you to be more careful, your latest changes still contain bugs related to this.

Reply:
Hi @matedo1  Sorry, this was overridden during code rebase to the latest main branch. I will fix it.

Reply:
fixed in latest commit

Reply:
done